### PR TITLE
Release 0.4.0

### PR DESCRIPTION
**Pull request by PVince81**
Closes https://github.com/owncloud/customgroups/issues/170

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -6,23 +6,39 @@ The format is based on [Keep a Changelog](http://keepachangelog.com/en/1.0.0/).
 
 ## [Unreleased]
 
+## [0.4.0] - 2018-12-03
+
+## Changed
```
Comment (davitol) [inline CHANGELOG.md]: ### Change ?

File: CHANGELOG.md [hunk 2]
```diff
@@ -6,23 +6,39 @@ The format is based on [Keep a Changelog](http://keepachangelog.com/en/1.0.0/).
...
+
+- Set max version to 10 because core platform is switching to Semver
+
+## Fixed
```
Comment (davitol) [inline CHANGELOG.md]: ### Fixed ? with 3 #

File: CHANGELOG.md [hunk 3]
```diff
@@ -6,23 +6,39 @@ The format is based on [Keep a Changelog](http://keepachangelog.com/en/1.0.0/).
...
+
+- Sort groups when requested from DB, fixes Oracle - [#187](https://github.com/owncloud/customgroups/issues/187)
+- Fix double encoding when displaying group name to delete - [#165](https://github.com/owncloud/customgroups/pull/165)
+- PHP 7.2 support - [#164](https://github.com/owncloud/customgroups/pull/164)
+
 ## [0.3.6] - 2018-01-11
 
 ## Fixed
```
Comment (davitol) [inline CHANGELOG.md]: same 3 # here

File: CHANGELOG.md (hunk 1 above)
Comment (PVince81) [reply CHANGELOG.md]: use the triple quotes power

**Comment by PVince81** [comment]
@davitol fixed

**Pull request merged**